libtextsecure/message_receiver.js

```diff
@@ -1028,7 +1028,7 @@ MessageReceiver.prototype.extend({
             );
 
             // Update the conversation
-            await conversation.onFriendRequestAccepted();
+            await conversation.onFriendRequestAccepted(true);
```

Comment:
Would this be clearer?
`await conversation.onFriendRequestAccepted({ updateUnread: true });` ?
Reading the code, the `true` might be misleadingly point to the acceptance or rejection value ;)